src/gatsby/src/components/park/parkDates.js

```diff
@@ -164,6 +171,37 @@ export default function ParkDates({ data, parkOperation }) {
       Object.values(openAccordions).every((isOpen) => isOpen)
   }, [openAccordions, subAreas.length])
 
+  const checkHash = useCallback(() => {
+    // Check hash in url
+    // if we find a matching parkSubArea, open that subArea accordion
+    let h = ""
+    let idx = 0
+    if (typeof window !== "undefined") {
+      h = window.location.hash
+      if (h !== undefined && h !== hash) {
+        subAreas.forEach(subArea => {
```

Comment:
If it's only searching for one subArea, maybe we should use a find/findIndex instead of a forEach, so it doesn't keep looping unnecessarily after it finds the subArea? (Or is it possible to find multiple matching subAreas?)